relay-event-normalization/src/normalize/mod.rs

```diff
@@ -1021,6 +1022,8 @@ pub fn light_normalize_event(
         // Check for required and non-empty values
         schema::SchemaProcessor.process_event(event, meta, ProcessingState::root())?;
 
+        TimestampProcessor.process_event(event, meta, ProcessingState::root())?;
```

Comment:
The timestamp processor runs before the clock drift processor. With this PR, Relay will drop events created before and coming from systems with a date before January 1, 1970. I believe it's acceptable to assume no systems will have their date set to a value prior to 1970.